### PR TITLE
[Api][Web][SIMS#443]Document Number

### PR DESCRIPTION
**Pull request by henryoforeh-aot**
I created migration file to alter document number column to accept null data

### CODE REVIEW

**Review by ann-aot — APPROVED** [review]
LGTM @henryoforeh-aot 👍

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by dheepak-aot — APPROVED** [review]
LGTM @henryoforeh-aot

File: sources/packages/api/src-sql/DisbursementSchedules/Alter-document-number-col.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
```
Comment (guru-aot) [inline sources/packages/api/src-sql/DisbursementSchedules/Alter-document-number-col.sql:1]: Can we please add some comments at the top ?
--Drop the NOT NULL constraint on document_number. Similarly to the rollback file too.

File: sources/packages/api/src/database/entities/disbursement-schedule.model.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export class DisbursementSchedule extends RecordDataModel {
    */
   @Column({
     name: "document_number",
-    nullable: false,
+    nullable: true,
   })
   documentNumber: number;
```
Comment (guru-aot) [inline sources/packages/api/src/database/entities/disbursement-schedule.model.ts:31]: please add ?  `documentNumber?: number;`, as it can be nullable now.

Comment (henryoforeh-aot) [reply sources/packages/api/src/database/entities/disbursement-schedule.model.ts:31]: I thought with ? it means the attribute can be an optional field and also undefined

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @henryoforeh-aot

**Review by abschwenker — APPROVED** [review]

**Pull request merged**